.gitignore

```diff
@@ -42,3 +42,5 @@ rpm/SOURCES/
 rpm/BUILDROOT/
 rpm/src/
 
+*.pyc
+properties.json
```

Comment:
properties.json seems to be a very general file to be ignored in general... I would need to understand why this is neded. Maybe through skype discussion.